include/zephyr/devicetree.h

```diff
@@ -4189,8 +4189,12 @@
  *     DT_ANY_INST_HAS_PROP_STATUS_OKAY(baz) // 0
  * @endcode
  */
+#define DT_ANY_INST_HAS_PROP_STATUS_OKAY__(idx, prop)	\
+	COND_CODE_1(DT_INST_NODE_HAS_PROP(idx, prop), (1,), ())
+#define DT_ANY_INST_HAS_PROP_STATUS_OKAY_(prop)	\
+	DT_INST_FOREACH_STATUS_OKAY_VARGS(DT_ANY_INST_HAS_PROP_STATUS_OKAY__, prop)
```

Comment:
this will break documentation for the macro below. please place internal macros at the bottom, under INTERNAL_HIDDEN

Reply:
Done